### PR TITLE
[MIG]migrate web_widget_image_webcam v10

### PR DESCRIPTION
**Pull request by smurf-U**

### CODE REVIEW

**Comment by smurf-U** [comment]
@oca-clabot  I have already signed up CLA & sent it to cla@odoo-community.org

**Comment by pedrobaeza** [comment]
The CLA is processed manually, so the bot will continue complaining until it will be verified.

File: web_widget_image_webcam/README.rst [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+<https://github.com/jhuckaby/webcamjs#important-note-for-chrome-47>`_.
+
+But, If you still want this module to work with websites without SSL / HTTPS.
+Here is the steps to do it easily (Always run in Adobe Flash fallback mode, but it is not desirable).
+
+Set the configuration parameter ``web_widget_image_webcam.flash_fallback_mode`` to ``1``
+
+Its done! Now this module also work with websites without SSL / HTTPS.
+
+Usage
```
Comment (elicoidal) [inline web_widget_image_webcam/README.rst]: Explicit or remove the section if not necessary

Comment (smurf-U) [reply web_widget_image_webcam/README.rst]: Excetly which section explicit or need to remove

Comment (smurf-U) [inline web_widget_image_webcam/README.rst]: Excetly which section explicit or need to remove

Comment (yajo) [reply web_widget_image_webcam/README.rst]: Those that are empty

Comment (smurf-U) [reply web_widget_image_webcam/README.rst]: Want to change usage​ to demo?

Comment (smurf-U) [reply web_widget_image_webcam/README.rst]: @Yajo and @elicoidal  i didn't get what are you try to say. so, can you more elaborate on what changes in this you need
Thanks..

Comment (elicoidal) [reply web_widget_image_webcam/README.rst]: The usage section contains no meaningful information (it is the standard boilerplate template): you can remove the section

**Comment by smurf-U** [comment]
@Yajo and @elicoidal Can you please review again...

**Review by elicoidal — APPROVED** [review]
LGTM Functional test OK!

Great contribution

**Review by yajo — APPROVED** [review]
Code & functional review

**Comment by lucode** [comment]
I had an issue with this module?
The error occurs if I try to open a Preference of a user. But after restarting the server and a new login it works now. Attention! when the error is active deinstallation is not possible, because the icon of the module does produce the same error.
`https://xxxxx.demo.koina.eu/web/content/3284-5c2a6f8/web.assets_common.js:2908
Zurückverfolgung
Error: QWeb2: Template 'WebCamDialog' not found
    at Object.exception (https://xxxxx.demo.koina.eu/web/content/3284-`

**Pull request merged**